pvnet_summation/data/datamodule.py

```diff
@@ -1,130 +1,81 @@
-""" Data module for pytorch lightning """
+"""Pytorch lightning datamodules for loading pre-saved samples and predictions."""
```

Comment:
Could you add a link to where the script is that make the samples? Hopefully that brings everything nicely together